e2e/workspace/module_test/runner.js

```diff
@@ -12,7 +12,7 @@ async function main() {
     return 1;
   }
 
-  if (result.status !== "finished") {
```

Comment:
what's the story here? status used to be reported "finished"?

Reply:
Yeah, that changed between cypress versions. I’ll leave a comment

Reply:
Does this imply we have some minimum cypress version that will work with the rule set?

Lets also move this to a separate PR since its not related to the "no-sandbox" tag